src/components/Task/Drawer/CommentList/CommentList.tsx

```diff
@@ -44,6 +46,12 @@ const TaskDrawerCommentList = ({
     },
   });
 
+  useEffect(() => {
```

Comment:
I may have misled you with the `useEffect` suggestion. It's fine but do you think `onError` https://www.apollographql.com/docs/react/api/react/hooks/#options would be better? Idk how easy that is to test. Hopefully it's easy.